Assets/HoloToolkit/Input/Scripts/Interactions/TapToPlace.cs

```diff
@@ -181,6 +195,18 @@ private void ToggleSpatialMesh()
             }
         }
 
+        private void CalculateColliderOffset()
```

Comment:
It would be handy to make this public as well, in case someone wants to manually update the offset themselves.